google.html

```diff
@@ -0,0 +1,67 @@
+<!-- get default html settings-->
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <title>Google JamesHoward1</title>
+
+    <link rel="stylesheet" href="styles/googleMain.css">
+
+    <!-- This is a comment, by the way -->
+
+</head>
+
+<body>
+<!-- Header
+line of anchor text links 
+Gmail
+Images
+Apps icon
+Sign In button
+-->
+<div id="header">
```

Comment:
For CSS, ensure you use classes rather than ids.  This is because they are not as flexible as classes and will cause problems further down the line